### DISCUSSION TITLE
When there are multiple players, is there a way to enqueue a new file?

### DISCUSSION THREAD

**Discussion by christopher-wilson** [smplayer-dev/smplayer#69] [Q&A] [answered]
I'm on Linux, KDE Neon to be specific, and we have an enqueue option in our context menu. If you are running multiple instances, this doesn't work, and just opens another instance.

Is it possible to enqueue a file in one of the currently running instances?

**Comment by smplayer-dev** [comment, marked as answer]
I'm afraid this is not possible. SMPlayer can only communicate with other instances when the option to use a single instance is enabled.

**Reply by christopher-wilson** [reply]
Thank you for the answer. I figured it was a longshot.
